test/integration/conftest.py

```diff
@@ -98,7 +98,7 @@ def db(mongo_client):
     raise RuntimeError("Can't find database")
 
 
-@pytest.fixture()
+@pytest.fixture(scope="class")
```

Comment:
I'm puzzled about 'class' scope here.  The docs describe this as meaning "the fixture is destroyed during teardown of the last test in the class", but these tests are not organized in a class..

Reply:
Ok, I see these fixtures used in other modules that organize tests in classes.

Reply:
With scope=class, if a bunch of tests are declared within a class, the fixture will be reused for all of them. Otherwise, a new fixture will be created.

There's a set of tests for downloading media, all declared in a class so we only have to upload things once. Conservator instance is shared across the download tests and not wiped between them.

Reply:
Doesn't change the behavior of standalone tests outside of a class